proprietary/assets/logo/amsterdam.svg

```diff
@@ -0,0 +1,6 @@
+<svg class="amsterdam-logo__image" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 115 40">
+  <title>Gemeente Amsterdam</title>
```

Comment:
Same comment as earlier here, the logo shouldn't be accessible for screen readers. Same for the other svgs

```suggestion
<svg class="amsterdam-logo__image" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 115 40">
```